sycl/source/half_type.cpp

```diff
@@ -9,8 +9,8 @@
 #include <CL/sycl/half_type.hpp>
 // This is included to enable __builtin_expect()
 #include <CL/sycl/detail/platform_util.hpp>
-#include <iostream>
 #include <cstring>
+#include <iostream>
```

Comment:
Why do you need this include?

Reply:
We don't need it anymore after the removal of the iostream operators.